src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+
+type Users = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+type Todo = {
+  id: number;
+  title: string;
+  userId: number;
+};
+
+type TodoInfoProps = {
+  todo: Todo;
+  users: Users[];
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo, users }) => {
+  return (
+    <article data-id={todo.id} className="TodoInfo TodoInfo--completed">
```

Comment:
You should check if the todo is completed to add or not the `--completed` modifier